### DISCUSSION TITLE
Repository Not Participating Hactoberfest

### DISCUSSION THREAD

**Discussion by Amankr200** [subhadipbhowmik/bio-branch#90] [Q&A] [answered]
is this Repository Not Participating in Hactoberfest @subhadipbhowmik 
![Screenshot 2024-10-07 224735](https://github.com/user-attachments/assets/5d7ab845-0e05-4cd5-a789-ccd354d1e01d)

**Comment by subhadipbhowmik** [comment, marked as answer]
Hi, thank you for your message. There was a typo in the topics of the project, but it has now been resolved. Rest assured, the labels were correct all along, so there’s no impact on contributions for Hacktoberfest, and all efforts are saved!

Best,
Shubhadip Bhowmik
Project Admin

<img width="479" alt="Screenshot 2024-10-07 231200" src="https://github.com/user-attachments/assets/de0544ce-a9fa-4b5d-a91d-0adff8e3bcec">


<img width="960" alt="Screenshot 2024-10-07 231412" src="https://github.com/user-attachments/assets/df39a54a-cd63-4985-902d-00bdd8c14763">

**Comment by subhadipbhowmik** [comment]
@Amankr200 We have solved the issue. Could you please confirm it?

**Reply by Amankr200** [reply]
yes problem is solved now
